pin-helper.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+PIN_COMMAND='
+set -e
+pip install -q -U pip pip-compile-multi
+pip-compile-multi -u \
+    -g base -g test -g dev -g docs
```

Comment:
Nit: should we pin `docs` on python 3.9 instead of 3.7?

Reply:
That was my initial thought. But once I talked to ahal, we decided to go with the minimum supported version of python, as long as it causes no problem to the version that rtd is running.